sdks/java/extensions/sql/build.gradle

```diff
@@ -79,6 +79,7 @@ dependencies {
   provided project(":sdks:java:io:kafka")
   provided project(":sdks:java:io:google-cloud-platform")
   compile project(":sdks:java:io:mongodb")
+  compile library.java.avro
```

Comment:
I suppose this is necessary because of the direct references to `org.apache.avro`? I think if we can push all that complexity into ParquetIO it won't be necessary, right?

Reply:
I was obliged to do this because of the strict dependency analysis enabled now forces to make use of dependencies explicitly in every module, but Avro in particular should not matter much remember that we leak Avro from `sdks/java/core` so this should not represent any issue.